### ISSUE TITLE
[DevOps] Deal with more secret scanning alerts.

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-team#80849]
## User Story
There are a bunch more secret scanning alerts opened up in another repository. They all look to be the same as in #80746, but that needs to be confirmed.

## Tasks
- [x] [DevOps] Deal with them.

## Acceptance Criteria

- [x] All alerts are confirmed to be for invalid and non-production tokens, or
- [x] Followup issue(s) opened to address any thornier problems.

**Comment by ndouglas** [comment]
@loripusey @anita-halici FYSA all 65 😩 of these are resolved now. (I completed this while blocked on other tickets.)
